articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,87 @@
+      <h1>Heading</h1>
+      <p>This paragraph has been styled using page level styling.</p>
+    </body>
+  </html>
+  ~~~
+
+  3. **External Styles** In this case, the styles used for the webpage are located in a completely different file. They are linked to the HTML file depending on which CSS file (Style) best fits the webpage. When developing a website with multiple pages this styling usually comes in handy.
+
+  ~~~html
+  <!DOCTYPE html>
+  <html>
+    <head>
+        <meta charset="utf-8">
+        <title>External Styles</title>
+        <link rel="stylesheet" type="text/css" href="css/myStyle.css"/>
+    </head>
+    <body>
+      <h1>Heading</h1>
+        <p>This is an example of External CSS</p>
+    </body>
+  </html>
+  ~~~
+  ~~~CSS
+  /*This is the CSS file called myStyle.css*/
+  body{
+    background-color: black;
+    color: white;
+  }
+  p{
+    color: purple;
+  }
+  ~~~
+
+### When to use each Style
+With these three options for styling in CSS, it becomes rather difficult to identify the most appropriate when building a webpage.
+
+  * _Local Style_ is not the first choice that comes to mind when dealing with CSS. At some point, it becomes too much of a burden to keep styling each element individually. By the time you come to the end of your project, the entire file is a mess. Even the smallest changes will require you to redo at least **70%** of the CSS. Which in the end prolongs the time needed to complete the project.
+
+  However, This does not mean that the style should not be used. It usually works well when you're trying to test out a new style on an element making it easier to determine if the style works well with the design or not. But always keep in mind that local styles have a higher priority over page-level and external styles. Meaning they will override any style property defined in page-level and external
+
+  * _Page Level Style_ is the clean version local style where you have everything located at one central place within the header of the HTML file. This gives the best results especially when you're in the development phase of the project. This is because browsers do not cache page-level styles like they do external styles.
+
+  * _External Styles_ gives you a wider range for its implementation. Since all the code used in the page styling is stored on a separate file it allows one to perform global changes on the project since one style sheet can control many pages. I prefer using external styling when it comes to launching the project or working with a live webpage. It is easier to upgrade since the design parameters of the entire site are defined in a single file.
+
+### Conclusion
+In the end of the what matters is how you apply the little knowledge you've gained. Whether it is testing out a new style on an element, building a simple single-page website, or building a company website that requires a uniform design from start to finish. Always consider the project, process, and product at all times.
```

Comment:
"In the end of the what matters is how you apply the little knowledge you've gained" => Remove, too vague, not helpful

"Whether it is testing out a new style on an element, building a simple single-page website, or building a company website that requires a uniform design from start to finish" => I see where you are going here. You may have completed the sentence with ", you might dabble in inline CSS, page-level styles or external styles, respectively"

While the scope is certainly one factor, the advantages and disadvantages you cite have more to do with hierarchy, keeping the code clean and modular, and time required to style multiple elements. So while you can briefly mention that scale is a factor, the conclusion should concisely summarize (in bullet points, if you like) the three main styles and their respective advantages and disadvantages